src/sentry/net/socket.py

```diff
@@ -3,7 +3,10 @@
 import six
 import ipaddress
 import socket
-from functools32 import lru_cache
+try:
+    from functools import lru_cache
+except ImportError:
```

Comment:
I agree that removing python2 support and related shin usage should be a separate block of work done after we get to py3.

Reply:
What I'm suggesting is that we can label these places as we create them so that it'll be easier to find and remove them when the time comes. It could just be a standard string that we can search for like `# TODO: Remove once we no longer support Python 2.7`. In my experience of upgrading libraries it makes it easier to find/remove unused branches like this.